### ISSUE TITLE
JCEF preview shows grey area after closing/opening/switching files

### ISSUE THREAD

**Issue by ahus1** [asciidoctor/asciidoctor-intellij-plugin#517]
When opening a AsciiDoc file, closing it, opening it again, and then switching between editors seems to close the JCEF browser, and the preview stays empty.

This ticket tries to provide a workaround and tracks the progress with the IDEA releases.

```
AsciiDoc plugin version 0.31.8
IntelliJ IDEA 2020.2 EAP (Ultimate Edition)
Build #IU-202.6109.22, built on July 1, 2020
Runtime version: 11.0.7+10-b944.16 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
```

**Comment by ahus1** [comment]
Issue created for IDEA project: https://youtrack.jetbrains.com/issue/IDEA-245183

**Comment by ahus1** [comment]
A workaround is available as part of 0.31.9

**Comment by ahus1** [comment]
JetBrains released the final version of 2020.2. The bug is still open and I can still reproduce the problem. Sad times.

The workaround seems to hold. I keep watching the issue in the JetBrains issue tracker, but close this issue for now.
